entrypoint.sh

```diff
@@ -22,9 +22,20 @@ if [[ "${HELM_DEBUG}" == "true" ]]; then
 	DEBUG_ARGS=" --debug"
 fi
 
-if [[ "${OPERATION}" == "deploy" ]]; then
+# approach 1: --state-values-set flag. This doesn't seem to propagate from helmfile.yaml to the imported release helmfile
+if [[ -n "$HELM_VALUES" ]]; then
+  HELM_VALUES_FLAG="--state-values-set ${HELM_VALUES}"
+fi
+
+# apprach 2: contents of a file to include in helmfile.yaml
+if [[ -n "$HELM_VALUES_YAML" ]]; then
+  echo -e "Using extra values:\n${HELM_VALUES_YAML}"
+  export HELM_VALUES_FILE="/tmp/extra_helm_values.yml"
+  echo "$HELM_VALUES_YAML" > "$HELM_VALUES_FILE"
+fi
```

Comment:
If this approach seems acceptable, I can ditch the `--state-values-set` approach since it's not working. I had liked it originally since (1) it was evident in the command what was being injected, and (2) by nature of being a command line flag it more strongly implied being a temporary override, rather than an alternative to declaring values in the actual helmfile / charts themselves.

In any case, wanted to run both potential approached by maintainers.